### ISSUE TITLE
Does allowElements block or drop elements?

### ISSUE THREAD

**Issue by Sora2455** [WICG/sanitizer-api#170]
According to this note from the spec:

> Note: allowElements creates a sanitizer that defaults to dropping elements, while blockElements and dropElements defaults to keeping unknown elements. Using both types is possible, but is probably of little practical use. The same applies to allowAttributes and dropAttributes.

Using only allowElements creates a sanitizer that drops elements. However, the example after it shows it instead _blocking_ elements, which is what Chrome's implementation does.

```js
const sample = to_node("Some text <b><i>with</i></b> <blink>tags</blink>.");

// Some text <b>with</b> text tags.
new Sanitizer({allowElements: [ "b" ]}).sanitize(sample);
```

**Comment by otherdaniel** [comment]
I agree; that should read "blocking", not "dropping". (I think the behaviour is consistent with how the spec defines it in the algorithms sections, specifically " sanitize action for an element" step 9.)

**Comment by annevk** [comment]
This seems like an editorial matter that might be naturally resolved as we move towards the new names.

**Comment by mozfreddyb** [comment]
Agreed. Can we close this?

**Comment by annevk** [comment]
Presumably someone still needs to rewrite the quoted text?

**Comment by mozfreddyb** [comment]
The quoted text no longer exists.
